plugins/SmfImport/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ INCLUDE(BuildPlugin)
 LINK_DIRECTORIES(${DRUMSTICK_FILE_LIBDIR})
 INCLUDE_DIRECTORIES(${DRUMSTICK_FILE_INCLUDEDIR})
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fexceptions")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fexceptions -ldrumstick-file -Wno-reorder")
```

Comment:
I have reverted to your commit and just fix that cannot link to drumstick-file problem, just see my new commit